### ISSUE TITLE
Only provide DNS64-generated responses to some clients

### ISSUE THREAD

**Issue by romanrm** [NLnetLabs/unbound#462]
Hello,

There seems to be no way to limit serving of DNS64-generated AAAA records only to certain subnets or IPs of requesting clients. A lot of per-client features exist, such as access-control, views and tags, but none of these were made with DNS64 in mind. This is a very significant limitation for me which totally precludes deployment of NAT64 and DNS64 with Unbound at the moment. Unless I'm missing something of course (which I'd love to). Is there any workaround to make this work at the moment, short of running multiple unbound processes?

Thanks

**Comment by wcawijngaards** [comment]
No, there is no such workaround. It requires a cache per view to keep the generated AAAA records in that cache. And this is not something we have.

Running different unbound server processes with the different settings and then for example use firewall rules to route the requests to their unbound server.

**Comment by romanrm** [comment]
Not familiar with the unbound code, but I hope this can be implemented some time in the future. Just an idea -- perhaps it could be done without requiring drastic changes to the code, say, keeping the unified cache for all views, and just filtering out AAAAs that match the configured DNS64 prefix before sending the response to clients of a certain access-control condition (e.g. "deny_dns64"). Thanks
